play-testkit/src/main/java/akka/grpc/play/JavaAkkaGrpcClientHelpers.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (C) 2018 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.grpc.play;
+
+import akka.actor.ActorSystem;
+import akka.grpc.GrpcClientSettings;
+import akka.grpc.GrpcClientSettings$;
+
+import play.api.test.NewTestServer;
+import play.api.test.ServerEndpoint;
+import play.api.test.ServerEndpoints;
+
+import javax.net.ssl.SSLContext;
+
+public final class JavaAkkaGrpcClientHelpers {
+  private static final GrpcClientSettings$ GrpcClientSettings = GrpcClientSettings$.MODULE$;
+
+  private JavaAkkaGrpcClientHelpers() {}
+
+  public static GrpcClientSettings grpcClientSettings(final NewTestServer testServer) {
+    final ServerEndpoint http2Endpoint = unsafeGetHttp2Endpoint(testServer.endpoints());
+    return grpcClientSettings(http2Endpoint, testServer.testServer().application().actorSystem());
+  }
+
+  public static ServerEndpoint unsafeGetHttp2Endpoint(final ServerEndpoints serverEndpoints) {
+    final scala.collection.Traversable<ServerEndpoint> possibleEndpoints =
+        serverEndpoints.endpoints().filter(e->e.scheme().equals("https") && e.httpVersions().contains("2"));
```

Comment:
Technically, gRPC (and HTTP/2) don't require `https` so this filter should only assert `e.httpVersions().contains("2")`.

Reply:
Ah, I see this is a refactor from pre-existing code. We can address separately when we get there.